data/alexandria start.txt

```diff
@@ -0,0 +1,1290 @@
+			`	"Are you ready to go, Captain <last> <first>?"`
+			choice
+				`	"I have some questions first."`
+					goto questions
+				`	"Let's go."`
+					goto leave
+			label questions
+			`	"What do you want to know?"`
+			choice
+				`	"What was in the book?"`
+					goto contents
+				`	"How will you find the next book?"`
+					goto seek
+				`	"Who was that monk, and where did he go?"`
+					goto monk
+				`	"No more questions. Let's go."`
+					goto leave
+				`	"I'm done with your quest. Enjoy the book."`
+					decline
+			label contents
+			`	"The book begins a story about a merchant, HoD Qav of Clan Botolphstone, destined to be the greatest warrior of the era. Through hard work, HoD Qav left the lands of Botolphstone, surviving on dangerous roads through wit alone. HoD Qav became an explorer, experiencing the vast continent on which their clan resides."`
+			`	"Why do you say 'their' clan instead of his or her?"`
+			`	"The opera doesn't specify the gender of the merchant. The way it's written, the merchant could be male or female... or both. Or neither, I suppose."`
+			`	"What happens next in the story?"`
+			`	Daemos looks at you as if you were a child. "To know what happens next, we'll have to find the second book: the Paq cha'DIch."`
+				goto questions
+			label seek
+			`	"My research suggests that the next book deals with war, unfairness, and greed. It is filled with mystery and amorality. That is also the very definition of the Syndicate. Their Director of Manufacturing is famous for his vast library."`
+				goto questions
+			label monk
+			`	"The monk? What monk? Haven't you ever seen a hologram before?"`
+				goto questions
+			label leave
+			`	"Excellent. We're going to Tinker in the Diphda system. It's at the edge of Syndicate space. Some stars around here are haunted by pirates. Take a route through the Paradise Planets and then Sol."`
+				accept
+	on visit
+		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
+
+
+
+
+mission "Alexandria Library Card: Second Volume: Step 2"
+	name "Alexandria Library Card"
+	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
+	blocked `Daemos complains, "I can't sleep in cooling ducts. I need a bunk and space for 1 ton of cargo."`
+	source Tinker
+	destination Amazon
+	landing
+	passengers 1
+	cargo "old books and a thousand pencils" 1
+	to offer
+		has "Alexandria Library Card: Second Volume: Step 1: done"
+	on offer
+		conversation
+			`You are led to a vast skyscraper overlooking a mountain range. From the glass-walled elevator, you take in the breathtaking view. The elevator stops on the top floor and you see a well-dressed man surrounded by bookshelves. He sitting on a gold-lined leather chair, pulled up to a marble desk. The bookshelves also appear to be hand-carved marble. Whoever this is, he is exceptionally rich.`
+			choice
+				`	(Interrupt him.)`
```

Comment:
The player is along for the ride, so conversation options with characters won't happen without good reason. The "choice" block gives them a break in the conversation so there is an abrupt pause between the surprise at the contents of the room, and the more mundane conversation that follows. That ensures they notice the expense of the surroundings rather than skipping to the next paragraph.